static/js/search.js

```diff
@@ -1161,6 +1161,31 @@ function populateResults(data, full, jumpToSingle) {
       div.textContent = `Number of results: ${count} (maximum is around 4000)`;
       header.append(div);
     }
+
+    // Detect "definitions exist but no uses"
+    let hasDefinitions = false;
+    let hasUses = false;
+
+    for (const [_, counts] of tupledCounts.entries()) {
+      if (_.includes("Definition") && counts.files > 0) {
```

Comment:
the "underscore" variable name means "we don't care".
If we're using it, we should name it